conditional-signing/browser/src/connect.ts

```diff
@@ -0,0 +1,186 @@
+    } else {
+      console.log(
+        `ℹ️  Using provided Capacity Credit with ID: ${LIT_CAPACITY_CREDIT_TOKEN_ID}`
+      );
+    }
+
+    console.log("🔄 Creating capacityDelegationAuthSig...");
+    const { capacityDelegationAuthSig } =
+      await litNodeClient.createCapacityDelegationAuthSig({
+        dAppOwnerWallet: ethersSigner,
+        capacityTokenId,
+        delegateeAddresses: [await ethersSigner.getAddress()],
+        uses: "1",
+      });
+    console.log("✅ Capacity Delegation Auth Sig created");
+
+    console.log("🔄 Executing Lit Action...");
+    const litActionSignatures = await litNodeClient.executeJs({
+      sessionSigs: await litNodeClient.getSessionSigs({
+        chain: CHAIN_TO_CHECK_CONDITION_ON,
+        capabilityAuthSigs: [capacityDelegationAuthSig],
+        expiration: new Date(Date.now() + 1000 * 60 * 60 * 24).toISOString(), // 24 hours
+        resourceAbilityRequests: [
+          {
+            resource: new LitPKPResource("*"),
+            ability: LitAbility.PKPSigning,
+          },
+          {
+            resource: new LitActionResource("*"),
+            ability: LitAbility.LitActionExecution,
+          },
+        ],
+        authNeededCallback: async ({
+          resourceAbilityRequests,
+          expiration,
+          uri,
+        }) => {
+          const toSign = await createSiweMessageWithRecaps({
+            uri: uri!,
+            expiration: expiration!, // 24 hours
+            resources: resourceAbilityRequests!,
+            walletAddress: await ethersSigner.getAddress(),
+            nonce: await litNodeClient.getLatestBlockhash(),
+            litNodeClient: litNodeClient,
+          });
+
+          return await generateAuthSig({
+            signer: ethersSigner,
+            toSign,
+          });
+        },
+      }),
+      code: litActionCode,
+      jsParams: {
+        conditions: [
+          {
+            conditionType: "evmBasic",
+            contractAddress: "",
+            standardContractType: "",
+            chain: CHAIN_TO_CHECK_CONDITION_ON,
+            method: "eth_getBalance",
+            parameters: [":userAddress", "latest"],
+            returnValueTest: {
+              comparator: ">=",
+              value: "1",
+            },
+          },
+        ],
+        authSig: await (async () => {
+          const toSign = await createSiweMessageWithRecaps({
+            uri: "http://localhost",
+            expiration: new Date(Date.now() + 1000 * 60 * 60 * 24).toISOString(), // 24 hours
+            walletAddress: await ethersSigner.getAddress(),
+            nonce: await litNodeClient.getLatestBlockhash(),
+            resources: [
+              {
+                resource: new LitPKPResource("*"),
+                ability: LitAbility.PKPSigning,
+              },
+              {
+                resource: new LitActionResource("*"),
+                ability: LitAbility.LitActionExecution,
+              },
+            ],
```

Comment:
```suggestion
            resources: [],
```
I don't think you need to provide any permissions for this sig since we're just using it to authenticate an address

Reply:
I think Lit Action execution is required